### ISSUE TITLE
Navbar icons fail WCAG AAA

### ISSUE THREAD

**Issue by koderjoker** [internetarchive/iaux#26]
**Is your feature request related to a problem? Please describe.**
The following issue aims at making the navbar more accessible.
The navbar icons for media and upload (class: iconochive-*) currently fail WebAIM's colour contrast check for WCAG AAA.

<img src="https://user-images.githubusercontent.com/30471843/54923388-22a87280-4f30-11e9-8fdb-64f34a7cb7d8.png"  width="200"/> <img src="https://user-images.githubusercontent.com/30471843/54923397-26d49000-4f30-11e9-8a4b-986f2b138e49.png"  width="40"/>

https://webaim.org/resources/contrastchecker/?fcolor=999999&bcolor=222222

**Describe the solution you'd like**
To choose a lighter colour for the navbar's icons.

**Comment by pezvi** [comment]
Another great issue, which is part of the navbar work being done internally. Claiming this issue, too.
